syncstorage/src/web/extractors.rs

```diff
@@ -446,20 +446,25 @@ impl FromRequest for BsoBody {
                 .unwrap_or_default()
                 > max_payload_size
             {
-                let err: ApiError = ValidationErrorKind::FromDetails(
+                return Err(ValidationErrorKind::FromDetails(
                     "payload too large".to_owned(),
                     RequestErrorLocation::Body,
                     Some("bso".to_owned()),
                     label!("request.validate.payload_too_large"),
                 )
-                .into();
-                return Err(err.into());
+                .into());
             }
             if let Err(e) = bso.validate() {
-                let err: ApiError =
-                    ValidationErrorKind::FromValidationErrors(e, RequestErrorLocation::Body, None)
-                        .into();
-                return Err(err.into());
+                return Err(ValidationErrorKind::FromValidationErrors(
+                    e,
+                    RequestErrorLocation::Body,
+                    None,
+                )
+                .into());
+                // let err: ApiError =
```

Comment:
drop the commented code?

Reply:
Yup @jrconlin , that's a WIP from a recent commit as I'm working on refactoring, had that on my list to remove. Getting my head around the two types of implementations and comparing syntax. Thx!